### DISCUSSION TITLE
Not an issue GS pro n64

### DISCUSSION THREAD

**Discussion by Nes-player4Life** [sanni/cartreader#361] [General Chat]
I have a few gameshark pros and non pros i have ripped. How can i help this project.
I also have an a action replay ems and xplorer64 im Looking to dump.

**Comment by Nes-player4Life** [comment]
Verizon of chips are:
9902 EA 3.2 pro
9907 EA 3.2 pro
9948 EA (fully working parallel port) Green led.

**Comment by majorpbx** [comment]
Only thing I need to add support is the ROM chip information. Now I know already that Xplorer64 might be an issue but it has the same ROM chips as some GameShark versions. That aside if it does not dump it might be an issue with the CPU on the board as Xplorer64 is different from what I've found on other carts.

**Reply by Nes-player4Life** [reply]
What do you refer to gameshark rom chip information as?

Xplore64 is quite larger than the chip found on the game sharks/action replay.
